### ISSUE TITLE
When I try to run i get "AnimateCC: AdobeAn dependency not found"

### ISSUE THREAD

**Issue by jerolaro** [bibixx/react-adobe-animate#52]

**Comment by juliandm** [comment]
Same here, it's hard to get this working with webpack

**Comment by bibixx** [comment]
@jerolaro @juliandm 🤔 could you try installing 2.4.0 version?

**Comment by mike-keogh** [comment]
Hey @jerolaro, @juliandm. I was having this issue whilst trying to use Gatsby. I managed to solve it by configuring the scripts to load before the page was rendering. If you are using a framework like Gatsby then that might help? I used this api to load the animation script and the createjs script ... https://www.gatsbyjs.com/docs/reference/config-files/gatsby-ssr/#onRenderBody

**Comment by bibixx** [comment]
Yes unfortunately this package is being limited by Adobe Animate's output file shortcomings. Not much can be done to this – ie. you have to load those file (createjs and animation script) before your react code is run
